lxd/daemon.go

```diff
@@ -395,30 +394,28 @@ func (d *Daemon) Init() error {
 		return err
 	}
 
-	if !d.os.MockMode {
-		/* Read the storage pools */
-		err = SetupStorageDriver(d.State(), false)
-		if err != nil {
-			return err
-		}
+	/* Read the storage pools */
+	err = SetupStorageDriver(d.State(), false)
+	if err != nil {
+		return err
+	}
 
-		/* Apply all patches */
-		err = patchesApplyAll(d)
-		if err != nil {
-			return err
-		}
+	/* Apply all patches */
+	err = patchesApplyAll(d)
+	if err != nil {
+		return err
+	}
 
-		/* Setup the networks */
-		err = networkStartup(d.State())
-		if err != nil {
-			return err
-		}
+	/* Setup the networks */
+	err = networkStartup(d.State()) // No-op if MockMode is on.
```

Comment:
I don't think we need to have the No-op comment here, if there's a reason for something to need to be under a MockMode check, then we should have a comment saying why, otherwise, we can assume all is fine.

Reply:
Fixed in 0d62d4fbef6355d16f935fe0f84efdee97a61a20.